dizoo/sokoban/envs/sokoban_env.py

```diff
@@ -0,0 +1,103 @@
+import gym
```

Comment:
add this env into [readme env list](https://github.com/opendilab/DI-engine#environment-versatility)